src/data_gradients/managers/abstract_manager.py

```diff
@@ -95,38 +97,88 @@ def execute(self):
             "computer vision datasets. click here: https://hubs.ly/Q01XpHBT0"
         )
 
-        datasets_tqdm = tqdm(
-            zip_longest(self.train_samples_iterator, self.val_samples_iterator, fillvalue=None),
-            desc="Analyzing... ",
-            total=self.n_batches,
-        )
+        # datasets_tqdm = iter(tqdm(
```

Comment:
please remove commented out code

Reply:
Fixed in e8f3e923454842e4a4f6c5446984f6c4100b2e67